src/components/Person/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "../../App.css";
+import { useNavigate } from "react-router-dom";
+import Avatar from "../../img/Avatar.png";
+
+const Person = ({ person }) => {
+  const navigate = useNavigate();
+
+  return (
+    <div key={person.id} onClick={() => navigate(`/person/${person.id}`)}>
+      <img
+        className="Person"
+        src={
+          person.profile_path
+            ? `https://image.tmdb.org/t/p/w500/${person.profile_path}`
```

Comment:
Since you are using `https://image.tmdb.org/t/p/w500/` multiple times in different files, you can create a values.js file and put it as:

const imageIconUrl = https://image.tmdb.org/t/p/w500/;

and then export it and import it in the needed files